manifests/h/Hashicorp/Vagrant/2.3.3/Hashicorp.Vagrant.installer.yaml

```
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Hashicorp.Vagrant
+PackageVersion: 2.3.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+- 1706
+UpgradeBehavior: install
+InstallLocationRequired: true
+RequireExplicitUpgrade: true
+DisplayInstallWarnings: false
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://releases.hashicorp.com/vagrant/2.3.3/vagrant_2.3.3_windows_amd64.msi
+  InstallerSha256: DE268C1D580066E23F6E334E3EA14850CD4AEB186E545AF08B1F7217FF67E6E7
+  ProductCode: '{608AAE8F-FAA0-4181-AB93-EA9AAEE0ED6E}'
+- InstallerLocale: en-US
+  Architecture: x86

```

```suggestion
- Architecture: x86
```
Is the installer specific to only `en-US` locale? Previous manifest's metadata show that maybe this is not required.